src/parse/state/tag.ts

```diff
@@ -376,6 +368,15 @@ function read_attribute(parser: Parser, unique_names: Set<string>) {
 	const colon_index = name.indexOf(':');
 	const type = colon_index !== -1 && get_directive_type(name.slice(0, colon_index));
 
+	if (unique_names.has(name) && type != "EventHandler") {
```

Comment:
```suggestion
	if (unique_names.has(name) && type !== "EventHandler") {
```

Not that it would really matter here, but we are trying to use non-coercion comparisons everywhere.